src/logic/currencyValues/store/middleware/index.js

```diff
@@ -0,0 +1,42 @@
+// @flow
```

Comment:
was this a requirement to save this to localStorage? I think that the value/rates are change pretty often so it's better to have up-to-date information

Reply:
@Agupane, indeed. What's the actual need to store it?